src/pybind/mgr/dashboard/controllers/nvmeof.py

```diff
@@ -365,8 +366,8 @@ def create(
             rbd_image_name: str,
             rbd_pool: str = "rbd",
             create_image: Optional[bool] = True,
-            size: Optional[int] = 1024,
-            rbd_image_size: Optional[int] = None,
+            size: Optional[int] = None,
+            rbd_image_size: Optional[Annotated[int, CephSizeBytes()]] = None,
```

Comment:
Nit: what about keeping it as a class, instead of instance (type hints are classes, although that's a pretty loose statement in Python)?
```suggestion
            rbd_image_size: Optional[Annotated[int, CephSizeBytes]] = None,
```
That will force to change `isinstance()` to `issubclass()`, and `@abstracmethod` [to `@abstractclassmethod`](https://docs.python.org/3/library/abc.html#abc.abstractclassmethod).

Reply:
I thought about it, but then I thought the current way will allow us to pass parameters to CephSizeBytes. for example in the future there might be a case where we only want to allow certain units or something, and we will not be abe to do so without instantiating an instance. It seems like this is the approach with the other arg types as well (as reflected from the parameters some of them receive in the ctor). wdyt?